src/VisualStudio/Core/Test/CodeModel/CSharp/EventCollectorTests.vb

```diff
@@ -786,7 +786,7 @@ class C
 </Code>
 
             Test(code, changedCode,
-                 ArgChange("System.CLSCompliant"))
+                 ArgChange("System.CLSCompliant", "foo"))
```

Comment:
I agree. Since we have a lot of them everywhere in our code it is would be better if we have a workitem to clean everything.